pkg/asset/machines/clusterapi.go

```diff
@@ -173,6 +175,37 @@ func GenerateClusterAPI(ctx context.Context, installConfig *installconfig.Instal
 		})
 	case azuretypes.Name:
 		// TODO: implement
+	case gcptypes.Name:
+		// Generate GCP master machines using ControPlane machinepool
+		mpool := defaultGCPMachinePoolPlatform(pool.Architecture)
+		mpool.Set(ic.Platform.GCP.DefaultMachinePlatform)
+		mpool.Set(pool.Platform.GCP)
+		pool.Platform.GCP = &mpool
+
+		gcpMachines, err := gcp.GenerateMachines(
+			installConfig,
+			clusterID.InfraID,
+			&pool,
+			string(*rhcosImage),
+		)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to create master machine objects")
```

Comment:
similar to other comment, this now returns just `error` and let's use `fmt.Errorf` instead of `errors.Wrap`